Makefile

```diff
@@ -177,7 +177,7 @@ deps-go:
 	go mod download
 ifeq ($(shell uname -s),Linux)
 	# Dependency required for Linux Fleet Desktop.
-	sudo apt-get install gcc libgtk-3-dev libayatana-appindicator3-dev -y
+	command -v apt-get && sudo apt-get install gcc libgtk-3-dev libayatana-appindicator3-dev -y || true
```

Comment:
I believe we could remove this `ifeq` from `deps-go`, and add this line on a separate `make deps-desktop-linux` target. I only added it because of the failing test in CI (ubuntu-linux).